src/main/java/eu/tsystems/mms/tic/testerra/plugins/xray/mapper/xray/XrayTestExecutionImport.java

```diff
@@ -414,17 +435,17 @@ public XrayTestExecutionImport(XrayTestExecutionIssue testExecutionIssue) {
         this.info.setSummary(testExecutionIssue.getSummary());
         this.info.version = testExecutionIssue.getFixVersions().stream().findFirst().map(JiraNameReference::getName).orElse(null);
         this.info.revision = testExecutionIssue.getRevision();
-        this.info.user = testExecutionIssue.getAssignee().getName();
+        this.info.user = testExecutionIssue.getAssignee() != null ? testExecutionIssue.getAssignee().getName() : null;
         this.info.startDate = testExecutionIssue.getStartDate();
         this.info.finishDate = testExecutionIssue.getFinishDate();
 
         List<String> testPlanKeys = testExecutionIssue.getTestPlanKeys();
-        if  (testPlanKeys.size() > 0) {
+        if  (testPlanKeys != null && testPlanKeys.size() > 0) {
             this.info.testPlanKey = testPlanKeys.get(0);
         }
 
         List<String> testEnvironments = testExecutionIssue.getTestEnvironments();
-        if (testEnvironments.size() > 0) {
+        if (testEnvironments != null && testEnvironments.size() > 0) {
```

Comment:
`XrayTestExecutionIssue#getTestEnvironments` calls `JiraIssue#getOrCreateFieldList` and cannot return `null`.

Reply:
agree, fixing now